db/migrations/postgresql/_init/V1.0.0.2__schemas.sql

```diff
@@ -0,0 +1,11 @@
+BEGIN;
+
+-- SCHEMAS WILL BE CREATED INSIDE OF THE CURRENTLY UED DATABASE;
+CREATE SCHEMA IF NOT EXISTS "app";
+CREATE SCHEMA IF NOT EXISTS "campaign";
+CREATE SCHEMA IF NOT EXISTS "util";
+
+-- DROP THE DEFAULT SCHEMA IF IT EXISTS
+DROP SCHEMA IF EXISTS "public";
```

Comment:
There are many db functions naturally existing in `public` schema. When we drop it we might miss those functions or types.

Reply:
Drop was removed, thanks.